src/assets/sampleBobaAPI.tsx

```diff
@@ -164,13 +161,13 @@ export const bobaList: BobaProps[] = [
     },
 ];
```

Comment:
create a separate jira for separating BobaList to read from backend/database

Reply:
tried and didnt work so decided to move on for now , can be revisit back later